src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="CommentList">
+    {comments.map(comment => (
+      <li>
+        <CommentInfo comment={comment} key={comment.id} />
+      </li>
```

Comment:
`key` should always go on the top level. In this case - in `li` tag.
```suggestion
      <li>
        <CommentInfo comment={comment} key={comment.id} />
      </li>
```

Reply:
Oops fixed it.